packages/core/src/codewhisperer/util/authUtil.ts

```diff
@@ -343,6 +346,30 @@ export class AuthUtil {
         }
     }
 
+    public async notifySessionConfiguration() {
+        const suppressId = 'amazonQSessionConfigurationMessage'
```

Comment:
This should be under typeof amazonQPromptName, see `type amazonQPromptName = keyof typeof amazonQPrompts`

Reply:
By doing so you don't have to do supressId as any

Reply:
right I forgot to remove any type, done